### DISCUSSION TITLE
Future Meeting Topics & Schedule

### DISCUSSION THREAD

**Discussion by pburkholder** [GSA-TTS/federal-platform-engineering-cop#12] [Ideas]
@derekmasaki @Chris-Lauer and I just met to discuss the schedule, and we're going to scale back to every-other-month for the summer, then resume monthly in September. So:

* June - No meeting
* July 24 - tentatively cloud.gov Page & how they've implemented OWASP ZAP and A11y  scans for their customer' static websites
* August - No meeting

(@brianrandallfox - sound OK?)

**Comment by brianrandallfox** [comment]
Sounds great!  Thank you all.

- Brian
720-663-0198
sent from my mobile device

**Comment by brianrandallfox** [comment]
I like the idea of a presentation on " cloud.gov Page & how they've implemented OWASP ZAP and A11y scans for their customer' static websites".  I will follow up with Andrew Burnes to see about his availability to present.  

Any other thoughts on future presentation topics?

**Comment by brianrandallfox** [comment]
I know that Omar Marrero was also interested in presenting on chaos engineering supporting COVID response efforts.  There was some interesting collaboration between Cloud.gov and USAF/Kessel Run to support this effort.  Omar is willing to participate from the KR side, who from Cloud.gov would like to potentially present with him?

**Comment by pburkholder** [comment]
I'll ask if we're OK talk about that collaboration

**Reply by pburkholder** [reply]
Ah, we are ok to talk. It's been published: https://www.gsa.gov/about-us/newsroom/news-releases/kessel-run-tts-collaborated-to-develop-capability-able-to-host-100-million-users-per-hour-with-cloudgov-03302022. 

I'm asking around at cloud.gov if anyone directly involved in the engineering on that can chime in. (I was busier on the compliance side for that effort).
